### ISSUE TITLE
Need help?

### ISSUE THREAD

**Issue by wuronin** [stolksdorf/homebrewery#137]
I noticed some minor issues with your curreny design that could be easily solved improving your nice tool for everybody.
- but the prefixed properties before the standard one.
The css could be reduced a lot simply by merging the encoded background image shared by:
> .phb hr+blockquote
 and
> .phb blockquote
also it would be useful to improve the support for other browsers by using a flex-based layout instead of the column base which is still experimental (and the .wide rules does not work in firefox)

If you want contact me and I will try to help.
Regards
Wu

**Comment by stolksdorf** [comment]
Thanks for your feedback

> put the prefixed properties before the standard one.

You're technically right, that the prefixed properties should come first. I might tweak them around but I highly doubt that it's causing any rendering issues.

> The css could be reduced a lot simply by merging the encoded background image shared by: > .phb hr+blockquote and > .phb blockquote 

I'm assuming you mean the `border-image`? If so the border image between those two rulesets is different so I can't merge them.

> it would be useful to improve the support for other browsers by using a flex-based layout instead of the column base which is still experimental (and the .wide rules does not work in firefox)

As far as I know there is no way to flow elements between two columns using css without using [CSS Columns](http://www.w3schools.com/css/css3_multiple_columns.asp). It would be impossible to make a flex-based layout without changing the way people write brews dramatically, which would require a massive migration of all exsisting brews and a loss in functionality IMO.

Honestly I'm not too concerned with supporting Firefox. The more time I spent trying to make existing features work on Firefox, the less time I have to create new features for everybody.

**Comment by wuronin** [comment]
I will have to check the encoded images again because I am almost sure that there is a repeated **border-image** somewhere (I found it inside the standalone css which could be from a previous release).
About firefox: it is getting every day a bit worse ... I discovered that your project work flawlessy with M$ Edge today (Even the print to pdf version...)
About everything else: You are right. Good luck with you project. And thanks for your efforts.
